### PR TITLE
Fix links titles

### PR DESCRIPTION
**Pull request by BigBlueHat**
@brianorwhatever thanks for these inputs! Review welcome. 😀

### CODE REVIEW

File: developers/wallets/native.md [hunk 1]
```diff
@@ -130,7 +129,30 @@ Example VCAPI protocol URL:
...
+`openid-credential-offer://?credential_offer=%7B%22credential_issuer%22%3A%22https%3A%2F%2Fqa.veresexchanger.dev%2Fexchangers%2Fz1A1GqykGBWKbwhFCDqFjMfnG%2Fexchanges%2Fz1A36rr6wEL25EEiikKvisVEC%22%2C%22credentials%22%3A%5B%7B%22format%22%3A%22ldp_vc%22%2C%22credential_definition%22%3A%7B%22%40context%22%3A%5B%22https%3A%2F%2Fwww.w3.org%2F2018%2Fcredentials%2Fv1%22%2C%22https%3A%2F%2Fpurl.imsglobal.org%2Fspec%2Fob%2Fv3p0%2Fcontext.json%22%5D%2C%22type%22%3A%5B%22VerifiableCredential%22%2C%22OpenBadgeCredential%22%5D%7D%7D%5D%2C%22grants%22%3A%7B%22urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Apre-authorized_code%22%3A%7B%22pre-authorized_code%22%3A%220065a8a0-069b-46f1-a857-4e1ce5047afd%22%7D%7D%7D`
```
Comment (brianorwhatever) [inline developers/wallets/native.md:133]: in the previous example we replaced the veresexchanger domain with an example.. They should probably match

**Review by brianorwhatever — APPROVED** [review]
Approving w/ caveat we match the examples

File: developers/wallets/native.md [hunk 2]
```diff
@@ -97,8 +97,7 @@ The endpoint declared there (`/switchboard` in the example above) will receive a
 Here is an example URL showing the `request` query parameter and its URL encoded value:
 
 <pre><code style="white-space: normal;overflow: auto;word-break: break-word;">
```
Comment (davidlehn) [inline developers/wallets/native.md:99]: `break-word` is apparently [deprecated](https://drafts.csswg.org/css-text/#word-break-property).  Perhaps dense strings like this should use `break-all`?

**Comment by BigBlueHat** [comment]
@brianorwhatever @davidlehn changes made.

**Comment by brianorwhatever** [comment]
Looks good!

**Review by brianorwhatever — APPROVED** [review]

**Review by davidlehn — APPROVED** [review]

**Pull request merged**